certinel.go

```diff
@@ -86,6 +99,17 @@ func (c *Certinel) Close() (err error) {
 	return err
 }
 
+// Wait blocks until the intial load of the key pair completes. If it returns
+// a nil error, GetCertificate and GetClientCertificate are ready for use.
+func (c *Certinel) Wait() error {
```

Comment:
How do you feel about Wait taking a context? I'm on the fence about it myself.

Reply:
I think taking a context is a good idea. I'll include it.